corehq/apps/users/migrations/0001_add_location_permission.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-09 02:05
+from __future__ import unicode_literals
+from corehq.sql_db.operations import HqRunPython
+from corehq.apps.users.management.commands.add_location_permission import update_all_roles
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    update_all_roles()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        HqRunPython(migrate),
```

Comment:
was thinking like this https://github.com/dimagi/commcare-hq/blob/8667a4159ad39d3307f92a3812d74006d610fd1b/corehq/form_processor/migrations/0037_get_form_by_id_fn.py, but that seems to be mostly for sharded dbs

I would say you could squash it but this doesn't have any other migrations. I wouldn't delete it in case someone makes a migration here later with the same name.

Maybe just an empty operations array? Not too worried about this because I don't think it'll grow quite to what accounting was